src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -278,6 +279,24 @@ def get_payment_context_data(submission: Submission) -> dict[str, Any]:
             "public_order_ids": submission.payments.get_completed_public_order_ids(),
         }
 
+    @staticmethod
+    def get_cosign_context_data(submission: Submission) -> dict[str, Any]:
+        if not submission.cosign_complete:
+            return {
+                "bsn": "",
+                "kvk": "",
+                "pseudo": "",
+                "date": "",
+            }
```

Comment:
maybe `return None` is better to signal that we can't say anything meaningful (because it's legacy co sign flow)?